includes/class-wc-form-handler.php

```diff
@@ -630,8 +631,26 @@ public static function order_again() {
 
 		do_action( 'woocommerce_ordered_again', $order->get_id() );
 
+		$num_items_in_cart = count( WC()->cart->get_cart() );
```

Comment:
Just a question; what happens here if the cart has items before you order again?